features/step_definitions/user_steps.rb

```diff
@@ -440,3 +441,25 @@ def page_heading_text(page)
   page.execute_script("document.getElementById('signingAlgorithm').value = '#{algorithm}';")
   click_on('I Agree')
 end
+
+Given('the user is at the {string} prompt page') do |idp|
+  idp_url = env('idps').fetch(idp)
+  prompt_page = URI.join(idp_url, 'start-prompt')
+  visit(prompt_page)
+end
+
+And('they select a service from the list') do
+  click_on('register for an identity profile')
+end
+
+And('they are sent to Test Rp') do
```

Comment:
Ditto, I think this should be a "Then" ... I believe you can still "And" them in the Gherkin script.